src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Figure figure;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int randomNumber = random.nextInt(5);

```

Comment:
maybe 100?)

Reply:
I dont think so, beacause i use this number to choose which figure will be created

Reply:
Okay use separete variable for figures